static/src/javascripts/projects/common/modules/experiments/tests/adblock-message.js

```diff
@@ -0,0 +1,75 @@
+define([
+    'common/utils/detect',
+    'common/utils/template',
+    'common/modules/ui/message',
+    'text!common/views/donot-use-adblock.html'
+], function (
+    detect,
+    template,
+    Message,
+    doNotUseAdblockTemplate
+) {
+    return function () {
+        this.id = 'AdBlockMessage';
+        this.start = '2015-03-27';
+        this.expiry = '2015-04-27';
+        this.author = 'Zofia Korcz';
+        this.description = 'Test if the users will disable adblock or at least click support link';
+        this.audience = 0.1;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Users will disable adblock or at least click support link';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = 'adblock message, hide, read more';
+        this.idealOutcome = 'Users will disable adblock on theguardian site';
```

Comment:
No, we are not. This is a bit tricky. We want to run this test to just check what will be the reaction from users, if they will click the link form the message or just hide it.

Of course we will also track users with adblock and see if their number within that period of time will decrease but it will not be necessary because of that test.